src/components/QuizGenerator.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import QuizContainer from './QuizContainer';
+import QuizGeneratorForm from './QuizGeneratorForm';
+
+function QuizGenerator() {
+  const [quizes, updateQuizes] = useState([]);
+
+  const updateQuizs = (quiz) => {
```

Comment:
This is not how this is done. Making intentional spelling errors so that names don't clash is incorrect. Instead use ->

const [quizes, _updateQuizes] = useState([])

const updateQuizes = (quiz) => {
    _updateQuizes([...quizes, quiz]);
}

Reply:
Done